test/e2e/testdata/compressed_yunikorn-configs.yaml

```diff
@@ -0,0 +1,33 @@
+#Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
space between # and the wording on all lines

Reply:
Other test data also need to add space to it. Do we want to fix this together in this PR?

https://github.com/apache/yunikorn-k8shim/blob/master/test/e2e/testdata/sleeppod_template.yaml